connectors/pom.xml

```diff
@@ -171,6 +171,59 @@
             <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
           </configuration>
         </plugin>
+
+
+        <!-- Disable some of these checks until we get the code base in basepom shape -->
+        <plugin>
+          <groupId>com.mycila</groupId>
+          <artifactId>license-maven-plugin</artifactId>
```

Comment:
This one should be relatively easy to keep, it has a `:format` goal to add the license headers.

Reply:
Lots of these files are also generated. And state don't edit me. So I wanted to skip the headache of addressing these at this time.